pkgs/by-name/ni/nixos-render-docs-redirects/src/nixos_render_docs_redirects/__init__.py

```diff
@@ -0,0 +1,86 @@
+import json
+import argparse
+from pathlib import Path
+
+
+def add_new_identifier(redirects: dict[str, list[str]], identifier: str, path: str) -> dict[str, list[str]]:
+    if anchor in redirects:
+        print("The identifier in the given element already exists.")
+        exit(1)
+    redirects[anchor] = [args.element]
+    return redirects
+
+
+def redirect_to_new_path(
+    redirects: dict[str, list[str]],
+    old_path: str,
+    old_anchor: str,
+    new_path: str,
+    new_anchor: str
```

Comment:
Do we really need the old path here? if we specify the identifier, all we would need to do is add the new location